docs/tutorials/ksql-basics-demo.rst

```diff
@@ -0,0 +1,311 @@
+.. _ksql_quickstart:
+
+Using KSQL to Write Streaming Queries Against Kafka Data
```

Comment:
Shouldn't we also cover the Docker variant here? I forgot what we discussed offline for the v2 in this regard.

Reply:
Filed JIRA https://confluentinc.atlassian.net/browse/DOCS-345